apps/hash-graph/Makefile.toml

```diff
@@ -60,4 +78,74 @@ run_task = { name = ["yarn-install-openapi", "graph-up", "test-rest-api", "graph
 [tasks.test-rest-api]
 # This is a temporary solution until we have e2e tests in place
 extend = "yarn"
-args = ["httpyac", "send", "--all", "${REPO_ROOT}/apps/hash-graph/hash_graph/tests/rest-test.http", "${@}"]
+args = ["httpyac", "send", "--all", "${REPO_ROOT}/apps/hash-graph/tests/rest-test.http", "${@}"]
+
+
+[tasks.yarn-install-openapi]
+description = "Installs the required yarn dependencies for generating the API client"
+extend = "yarn"
+cwd = "${REPO_ROOT}/apps/hash-graph/clients"
```

Comment:
tiny nit that's probably not worth addressing:
Instead of the `cwd` path here we could pass the workspace name of graph client generator (`@hashintel/hash-graph-client-generator`). Would make it so we don't need to specify the path here and potentially get it wrong in 2 places. But since we're extending yarn I'm not sure what's best

Reply:
Discussed offline, this will be addressed in a follow-up moving the clients into `libs`